dotnetv3/Kinesis/README.md

```diff
@@ -0,0 +1,72 @@
+# Amazon Kinesis code examples in C#
+
+## Purpose
+
+This folder contains examples that show how to use the AWS SDK for .NET 3.x to
+get started working with Kinesis features such as streams and consumers.
+
+With Amazon Kinesis, you can ingest real-time data such as video, audio, application logs, website clickstreams, and IoT telemetry data for machine learning, analytics, and other applications. Amazon Kinesis enables you to process and analyze data as it arrives and respond instantly instead of having to wait until all your data is collected before the processing can begin.
+
+## Code examples
+
+- [CreateStreamExample](CreateStreamExample/) - Create a new Kinesis stream.
+- [DeleteStreamExample](DeleteStreamExample/) - Delete an existing Kinesis stream.
+- [DeregisterConsumerExample](DeregisterConsumerExample/) - Deregister consumer from Kinesis stream.
+- [ListConsumersExample](ListConsumersExample/) - List the consumers of a Kinesis stream.
+- [ListStreamsExample](ListStreamsExample/) - List Kinesis data streams.
+- [ListTagsExample](ListTagsExample/) - List the tags associated with a Kinesis stream.
+- [RegisterConsumerExample](RegisterConsumerExample/) - Register a consumer to a Kinesis stream.
+- [TagStreamExample](TagStreamExample/) - Add tags to a Kinesis stream.
+
+## ⚠ Important
+- We recommend that you grant your code least privilege, or at most the minimum
+  permissions required to perform the task. For more information, see
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the AWS Identity and Access Management User Guide. 
```

Comment:
AWS Identity and Access Management -> AWS Identity and Access Management (IAM)

Reply:
I've had that line in all of my README files, you're the first one that caught that.